security/seccomp-profiles.adoc

```diff
@@ -7,13 +7,11 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 An {product-title} container or a pod runs a single application that performs one or more well-defined tasks. The application usually requires only a small subset of the underlying operating system kernel APIs.
-Seccomp, secure computing mode, is a  Linux kernel feature that can be used to limit the process running in a container to only call a subset of the available system calls. These system calls can be configured by creating a profile that is applied to a container or pod.
-Seccomp profiles are stored as JSON files on the disk.
+Seccomp, secure computing mode, is a  Linux kernel feature that can be used to limit the process running in a container to only call a subset of the available system calls. 
 
-[IMPORTANT]
-====
-OpenShift workloads run unconfined by default, without any seccomp profile applied.
-====
+The `restricted-v2` SCC applies to all newly created pods in {product-version}. The default seccomp profile `runtime/default` is applied to these pods.  
```

Comment:
This is not strictly correct. I would reword it to something like:

The `restricted-v2` SCC is the new SCC by default, if your workload doesn't have access to a different SCC, newly created pods in {product-version} will have its seccomp profile configured to `runtime/default` as mandated by the SCC.

Reply:
what is the situation with existing running pods or pods with access to a different SCC for example `restricted`. For these pods I assume if you want to have seccomp profile configured to runtime/default you need to update and that is what I am trying to show how to do here by adding the `restricted-v2` to the ServiceAccount. I am trying to work out how to set the seccomp profile to runtime/default for pods where it is not set already.

Reply:
As mentioned earlier we may need to have two sections now.

4.11 clusters that were upgraded from 4.10
4.11 fresh installed clusters

For the former, you're right. For the later we should reword as I suggested.